doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con-timer.adoc

```diff
@@ -3,6 +3,21 @@
 
 You can use timers to trigger logic after a certain period or to repeat specific actions at regular intervals. Timers wait for a predefined amount of time before triggering once or repeatedly.
 
+== Selecting the correct timer for your environment
+{PRODUCT} supports two types of timers:
+
+* Quartz: Recommended for use on Spring Boot and Tomcat
```

Comment:
Maybe with Spring Boot? You say 'with EAP'

Reply:
Agree. Done.